Moya.xcodeproj/project.pbxproj

```diff
@@ -302,23 +288,20 @@
 			isa = PBXHeadersBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				5B9281C0308BF58977D27DE9 /* ReactiveMoya.h in Headers */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 		9F6005B4252938375E1B61C9 /* Headers */ = {
 			isa = PBXHeadersBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				55D38822179474648606A08E /* RxMoya.h in Headers */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 		F4CA905CBCCD3A2D1D028529 /* Headers */ = {
 			isa = PBXHeadersBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				42414D9A4174BE3C8A63A2D4 /* Moya.h in Headers */,
```

Comment:
@sunshinejr CI fail is possibly related to this?
`no umbrella header found for target 'Moya', module map will not be generated`